traiders/frontend/public/index.html

```diff
@@ -24,7 +24,7 @@
       work correctly both with client-side routing and a non-root public URL.
       Learn how to configure a non-root public URL by running `npm run build`.
     -->
-    <title>Traiders</title>
+    <title>React App</title>
```

Comment:
I think you made this commit without pulling the last version of the master, try to pull master branch into your branch and then make another commit. The problem with this commit is it is trying to remove some changes done by other team members.

Reply:
It should be fixed